lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -682,6 +683,14 @@ balanceTotal =
         initBal = ApiT $ WalletBalance
             {available = Quantity v, Types.total = Quantity v}
 
+
+rewardBalance
+    :: HasField' "reward" s (Quantity "lovelace" Natural)
+    => Lens' s Natural
+rewardBalance = lens
+        (getQuantity . getField @"reward")
+        (\(s,v) -> setField @"reward" (Quantity v) s)
```

Comment:
**[sugestion]**

You may want to rebase on latest master, this lens now exists as `balanceReward`.